docs/install-azure-repos.md

```diff
@@ -0,0 +1,180 @@
+[Go back to the main documentation page](../README.md)
+# Installing Frogbot on Azure Repos repositories
+
+To install Frogbot on Azure Repos repositories:
+
+1. Go to Azure DevOps Pipelines of the relevant project, and add `New pipeline`.
```

Comment:
Go to Azure DevOps Pipelines of the relevant project, and add `New pipeline`.
-->
Go to your Azure Pipelines project, and add a new pipeline.

Let's also add the relevant image from the Azire DevOps UI.

Reply:
done